src/css/components/panel.scss

```diff
@@ -157,3 +156,23 @@ h4 + .panel {
   font-weight: 600;
   padding-bottom: $grid-1;
 }
+
+.panel-routes,
+.panel-services {
+
+  h1 {
+    font-weight: 400;
+  }
+
+  h3 {
+    font-size: 0.75rem;
+
+    a {
+      color: #056dd4;
+
+      .route-link {
```

Comment:
I wrote code recently so that all links in these panels will be underlined, so I think this is not needed.